pom.xml

```diff
@@ -159,6 +159,10 @@
         <artifactId>junit-jupiter</artifactId>
         <version>${junit.version}</version>
       </dependency>
+       <dependency>
+  <groupId>org.mockito</groupId>
+  <artifactId>mockito-core</artifactId>
```

Comment:
Version should be added here. By doing so we will have the same version automatically in all sub-modules